tests/integration/docker/mtu.bats

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bats
+# *-*- Mode: sh; sh-basic-offset: 8; indent-tabs-mode: nil -*-*
+
+#  This file is part of cc-oci-runtime.
+#
+#  Copyright (C) 2017 Intel Corporation
+#
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU General Public License
+#  as published by the Free Software Foundation; either version 2
+#  of the License, or (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, write to the Free Software
+#  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+# MTU testing using Swarm
+
+SRC="${BATS_TEST_DIRNAME}/../../lib/"
+# number of replicas to launch
+number_of_replicas=1
+# number of attempts
+number_of_attempts=5
+
+#This function will verify that swarm is not running or it finishes
+function clean_swarm_status() {
+	for j in `seq 0 $number_of_attempts`; do
+		if $DOCKER_EXE node ls; then
+			$DOCKER_EXE swarm leave --force
+			# docker swarm leave is not inmediately so it requires time to finish
+                        sleep 5
+                else
+			break
+		fi
+	done
+}
+
+
+setup() {
+	source $SRC/test-common.bash
+	runtime_docker
+	clean_swarm_status
+	interfaces=($(readlink /sys/class/net/* | grep -i pci | xargs basename -a))
+	swarm_interface_arg=""
+	for i in ${interfaces[@]}; do
+		if [ "`cat /sys/class/net/${i}/operstate`" == "up" ]; then
+			swarm_interface_arg="--advertise-addr ${i}"
+			break;
+		fi
+	done
+	$DOCKER_EXE swarm init ${swarm_interface_arg}
+	$DOCKER_EXE service create --name testswarm --replicas $number_of_replicas --publish 8080:80 nginx /bin/bash -c "hostname > /usr/share/nginx/html/hostname; nginx -g \"daemon off;\"" 2> /dev/null
+	while [ `$DOCKER_EXE ps --filter status=running --filter ancestor=nginx:latest -q | wc -l` -lt $number_of_replicas ]; do
+		sleep 1
+	done
+}
+
+@test "Checking MTU values in different interfaces" {
+	ip_address=$(mktemp)
+	container_id=`$DOCKER_EXE ps -qaf "name=testswarm"`
+	network_settings_file=`$DOCKER_EXE inspect $container_id | grep "SandboxKey" | cut -d ':' -f2 | cut -d '"' -f2`
+	[ -f $network_settings_file ]
+	nsenter --net=$network_settings_file ip a > "$ip_address"
+	#checking ceth0 interface
+	grep "ceth0" "$ip_address" | grep "mtu 1450"
```

Comment:
A couple of queries here:

- I'm not clear how you know this value should be `1450` but `eth1` has an MTU of `1500`? This doesn't appear to be set in this test? Could you add a comment to explain these figures please.
- I suggest you anchor this `grep` to make it safer. Something like:
  ```
  egrep "\<mtu 1450\>"
  ```
  (That will stop it matching a value like `mtu 1450999999`).